addon/globalPlugins/searchWith/__init__.py

```diff
@@ -439,12 +439,14 @@ def makeSettings(self, sizer):
 		self.availableLanguages = languageHandler.getAvailableLanguages(presentational=True)
 		googleChoices = [x[1] for x in self.availableLanguages]
 		deepLChoices = [x[1] for x in deepLLanguages]
-		sizer1= gui.guiHelper.BoxSizerHelper(self, orientation= wx.VERTICAL)
+		sizer1= gui.guiHelper.BoxSizerHelper(staticTranslationEnginesSizer.GetStaticBox(), orientation= wx.VERTICAL)
```

Comment:
It would be nice if we could change sizer1 to another name that makes more sense.